geniza/common/management/commands/visual_tests.py

```diff
@@ -42,7 +42,7 @@ def take_snapshots(self, browser):
         percy_snapshot(browser, "Document Search filter")
 
         # document search
-        browser.get("http://localhost:8000/documents/?q=the+writer&per_page=2")
+        browser.get("http://localhost:8000/documents/?q=the+writer+באנפנא&per_page=2")
```

Comment:
I wonder if we should have a query here that also shows the Hebrew being highlighted? Not strictly necessary though.

Reply:
Yeah, I was trying to get that but with the per-page results it didn't show up. Should I tweak the query and try to get it in the first two? I think we'll want to see it for testing fonts.

Reply:
Yes, I wonder if just using "באנפנא" as the first keyword would work? Might also be helpful to include an English word that's going to exist in the description of a document containing "באנפנא", then you could omit "writer" and we can see both English and Hebrew highlighted.